portfolio/src/main/webapp/comments.html

```diff
@@ -21,7 +21,20 @@
         </ul>
     </nav>
     <div id="container">
-        <div id="data-container"></div>
+        <div id="comment-container">
+            <h1>Leave your Comments Below</h1>
+            <form action="data" method="POST">
+                <label class="label">Name</label>
+                <input class="contact" type="text" name="name"></input>
+                <label class="label">Comment</label>
+                <input class="contact" type="text" name="comment"></input>
+                <input class="button" type="submit" />
+            </form>
+            <label id="quantity-label" for="quantity">Number of comments shown (-1 means that all comments are shown below):</label>
+            <input type="number" id="quantity" name="quantity" min="-1" value="-1" onchange="getComment();"/>
```

Comment:
Not a technical issue, but from the UX point of view, I would not choose an input box with a magic -1 input that needs to be explained. To avoid the -1, you can add a checkox to toggle whether to limit results at all, and only when checkbox is selected, enable the input value control. Some web sites also use a drop-down with pre-filled values like [all, 1, 5, 10, 25, 50], or something like that.